Formula/m/mariadb.rb

```diff
@@ -71,6 +73,8 @@ def install
 
     # Use brew groonga
     rm_r "storage/mroonga/vendor/groonga"
+    rm_r "extra/wolfssl"
+    rm_r "zlib"
```

Comment:
In theory, a no-op, but better to be safe and remove vendored SSL and Zlib. Others like PCRE2 and fmt are fetched by CMake. For now, we are using vendored fmt since we haven't updated - #176514, #178811